### DISCUSSION TITLE
今やってほしいこと書いてくれ～

### DISCUSSION THREAD

**Discussion by RainbowPuiPuiMolcar** [HamsterCrab/Science-PC_Club_Game#69] [Ideas]
書いたこと頑張ってやるから

**Comment by RainbowPuiPuiMolcar** [comment]


**Reply by Hamster-crab** [reply]
そんなことにAI使うとか

**Reply by Hamster-crab** [reply]
Pythonのhello,world書かせるのにAI使うようなもんだよ

**Comment by RainbowPuiPuiMolcar** [comment]
AIに関数作らせてみたらこうなったんだけど大丈夫そう？

**Reply by Hamster-crab** [reply]
舐めてんの？

**Reply by Hamster-crab** [reply]
まず関数の意味を理解しましょう

**Reply by Hamster-crab** [reply]
AIばっか使うな

**Reply by RainbowPuiPuiMolcar** [reply]
急に関数とか言われてわかるわけないでしょどこにどんな風に関数作れとか言わないと

**Reply by Hamster-crab** [reply]
磯貝から聞いてないの？

**Reply by RainbowPuiPuiMolcar** [reply]
どこにとかどんな風にとかは聞いてないね、

**Reply by Hamster-crab** [reply]
次から磯貝経由じゃなくてC++使いの林に直接言うわ
あとAIにコードまるまる送らないで

**Reply by Hamster-crab** [reply]
あともうそれは自分で書く

**Comment by Hamster-crab** [comment]
AIに丸ごとコード送るなクソ畜生

**Reply by RainbowPuiPuiMolcar** [reply]
私は初心者です。何をやっていいのか何をやった駄目なのか私にはわかりません。そこらへんを教えるのはあなたでしょ

**Reply by Hamster-crab** [reply]
前にもAIに丸ごとやっちゃだめだよとは言ったよね？
関数とは何かを調べたりするのはいいが、関数のような簡単なやつにAIを使うな

**Reply by Hamster-crab** [reply]
AIに丸ごとコードを送るなって何回言えばいいの？11451481019194545回？

**Reply by Hamster-crab** [reply]
あとなんの初心者？ヒカマニ？何？

**Reply by Hamster-crab** [reply]
何でもかんでもAIは良くないよ

**Comment by RainbowPuiPuiMolcar** [comment]
あのダメージ音ってあれでいいん？
さすがにはむすたーがあれはなぁ..って
